sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/aio/_client.py

```diff
@@ -470,6 +509,9 @@ async def update_key_properties(self, name: str, version: "Optional[str]" = None
         else:
             attributes = None
 
+        policy = kwargs.pop("release_policy", None)
+        if policy is not None:
+            policy = self._models.KeyReleasePolicy(content_type=policy.content_type, data=policy.data)
```

Comment:
Should this be passed to KeyUpdateParameters?

Reply:
It should be; thank you for noticing! I should probably add a test for this.